README.md

```diff
@@ -15,13 +15,26 @@ following:
   * FixNum, Boolean and other field values are not removed - type must respond
     to `:empty?` to be considered empty
 
-* Moves "undefined" values to a top level field called `undefined`
+* Has multiple ways to handle "undefined" fields - that is - fields that
+  are not listed in `default_keep_fields` or in `extra_keep_fields`
+  * If `use_undefined true`, then undefined top level fields are moved
+    to a top level field called `undefined`
+  * If `undefined_to_string true`, then the values of undefined top level
+    fields are converted to their JSON string representation
+  * If `undefined_dot_replace_char` is set to a string value, then top
+    level fields with a `'.'` in the field name will have the `.` changed
+    to a `'_'` (by default - replace char is configurable)
+  * If `undefined_max_num_fields` is a number greater than `-1`, if the
```

Comment:
additional "if"

Reply:
"If `undefined_max_num_fields` is a number greater than `-1`, when the number of undefined fields is greater than this number all ... "

Reply:
fixed wording in a slightly different way